src/NUnitEngine/nunit.engine.tests/Runners/MasterTestRunnerTests.cs

```diff
@@ -21,53 +21,131 @@
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 // ***********************************************************************
 
+using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
+using System.Reflection;
+using System.Text;
 using System.Xml;
 using NUnit.Framework;
 using NUnit.Tests.Assemblies;
-using System.Reflection;
-using System.IO;
-using NUnit.Tests;
 
 namespace NUnit.Engine.Runners.Tests
 {
-    [TestFixture(1)]
-    [TestFixture(2)]
+    using Services;
+    using Services.Tests.Fakes;
+
+    [TestFixtureSource("FixtureData")]
```

Comment:
Only as I'm asking for other changes anyway...how about `nameof(FixtureData)` - for extra future-refactor-safety? 😄